src/components/line-segment.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes } from "react";
```

Comment:
Nice factoring-out of a sub-component!

Can I haz a small, basic test case? :]